tests/e2e/default/reports/reports-subject.wdio-spec.js

```diff
@@ -0,0 +1,430 @@
+const utils = require('../../../utils');
+const placeFactory = require('../../../factories/cht/contacts/place');
+const userFactory = require('../../../factories/cht/users/users');
+const personFactory = require('../../../factories/cht/contacts/person');
+const loginPage = require('../../../page-objects/default/login/login.wdio.page');
+const commonPage = require('../../../page-objects/default/common/common.wdio.page');
+const reportsPage = require('../../../page-objects/default/reports/reports.wdio.page');
+const sentinelUtils = require('../../../utils/sentinel');
+const testReports = require('./test-reports');
+
+describe('Reports Subject', () => {
+  const waitElementTextEquals = async (elementGetter, expectedText) => await browser.waitUntil(async () =>
+    await elementGetter.getText() === expectedText);
+
+  const verifyReportContent = async ({
+    reportName: reportNameValue,
```

Comment:
Minor naming nitpick, but I think this should be `formName` instead of `reportName`.  Generally when I think of "reports" I just think of the documents, themselves, that are created by the form.